distributions/aws/aws-telemetry/job.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-kubelow-telemetry
+spec:
+  ttlSecondsAfterFinished: 0
+  backoffLimit: 3
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
is this necessary ?

Reply:
Yes, sidecar is a proxy that mediates inbound and outbound communication and since these jobs are standalone, we dont need it

Reply:
I mean to say should be remove it ?